apps/prairielearn/elements/pl-matching/pl-matching.py

```diff
@@ -85,7 +99,7 @@ def categorize_matches(element, data):
     children = element[:]
     children.sort(key=lambda child: child.tag)
 
-    def make_option(name, html):
+    def make_option(name: str, html: str) -> dict[str, int | str]:
```

Comment:
It'd be nice to eventually use some `TypedDict`s in here. Won't block on this.

Reply:
I think if we revise this element in the future, that will definitely be helpful.